pkg/snet/directtransport/client.go

```diff
@@ -0,0 +1,18 @@
+package directtransport
+
+import (
+	"context"
+
+	"github.com/SkycoinProject/dmsg/cipher"
+	"github.com/SkycoinProject/skycoin/src/util/logging"
+)
+
+// Client is a direct transport client.
+type Client interface { // TODO(nkryuchkov): use
```

Comment:
Should these todos be removed?

Reply:
@Darkren I started to use this interface in a branch that fixes #409 and removed the `TODO` there.